lib/modules/manager/bazel/artifacts.spec.ts

```diff
@@ -0,0 +1,554 @@
+import crypto from 'crypto';
+import { codeBlock } from 'common-tags';
+import * as httpMock from '../../../../test/http-mock';
+import { partial } from '../../../../test/util';
+import type { UpdateArtifact } from '../types';
+import { updateArtifacts } from './artifacts';
```

Comment:
```suggestion
import { updateArtifacts } from '.';
```
better code coverage for branches